packages/nuxt/src/Host.vue

```diff
@@ -91,33 +76,39 @@ const listen: ListenFunc = (listener) => {
   };
 };
 
-onBeforeMount(() => {
-  watchEffect(() => {
-    if (!cachedMount) {
-      loadScript(url)
-        .then(() => loadModule(name))
-        .then(({ default: createComposableApp }) => {
-          let remoteMount;
-          if (typeof createComposableApp === "function") {
-            const { mount } = createComposableApp({
-              isSelfHosted: false,
-            });
-            remoteMount = mount;
-          } else if (
-            typeof createComposableApp === "object" &&
-            createComposableApp.mount
-          ) {
-            remoteMount = createComposableApp.mount;
-          } else {
-            throw new Error("Remote module didn't return a function");
-          }
-          mount.value = remoteMount;
-          mountCache.set(mountKey, remoteMount);
-        })
-        .catch((err) => {
-          error.value = err;
+const HostWrapper = (mount: MountFunc, url?: string) => {
+  //@ts-ignore
+  return h(Mount, {
+    mount,
+    runtime,
+    basename,
+    listen,
+    setError: (e: any) => error.value = e,
+  });
+
```

Comment:
What IDE do yo use? we have prettierrc.js at the root of the repo. When I save this file on my computer it formats it differently

Reply:
VSCode - but I've installed [volar(https://github.com/johnsoncodehk/volar) which might be messing about with that formatting?

Reply:
Maybe, I guess you could disable Volar and `> Format Document` to check